.travis.yml

```diff
@@ -1,16 +1,16 @@
 language: generic
-dist: trusty
```

Comment:
So which one did it even use before if there's both entries.

Reply:
I'm pretty sure xenial, see #44079